mysite/missions/shell/tests.py

```diff
@@ -0,0 +1,242 @@
+# This file is part of OpenHatch.
```

Comment:
For the files you've created here, you can/should add a copyright line (ala https://github.com/openhatch/oh-mainline/blob/master/mysite/bugsets/models.py#L6 for instance)!